src/test/java/org/broadinstitute/hellbender/tools/exome/sexgenotyper/TargetCoverageSexGenotypeCalculatorUnitTest.java

```diff
@@ -58,34 +59,30 @@
     private static final long RANDOM_SEED = 1984;
     private static final double EPSILON = 1e-12;
 
-    private static final List<ContigGermlinePloidyAnnotation> HOMO_SAPIENS_GERMLINE_CONTIG_PLOIDY_ANNOTATIONS;
-    static {
-        HOMO_SAPIENS_GERMLINE_CONTIG_PLOIDY_ANNOTATIONS = ContigGermlinePloidyAnnotationTableReader
-                .readContigGermlinePloidyAnnotationsFromFile(TEST_CONTIG_PLOIDY_ANNOTS_FILE);
-    }
+    private static List<ContigGermlinePloidyAnnotation> HOMO_SAPIENS_GERMLINE_CONTIG_PLOIDY_ANNOTATIONS;
 
     private static final List<Target> RANDOM_TARGETS_WITH_BAIT_COUNTS = annotateTargetsWithRandomBaitCounts(
             generateRandomTargets(new Random(RANDOM_SEED)), new Random(RANDOM_SEED));
     private static final List<Target> RANDOM_TARGETS_WITHOUT_BAIT_COUNTS = generateRandomTargets(new Random(RANDOM_SEED));
 
-    private static final ReadCountCollection RANDOM_READ_COUNTS_WITH_BAIT_BIAS;
-    private static final List<String> TRUTH_SEX_GENOTYPES_WITH_BAIT_BIAS;
+    private static ReadCountCollection RANDOM_READ_COUNTS_WITH_BAIT_BIAS;
+    private static List<String> TRUTH_SEX_GENOTYPES_WITH_BAIT_BIAS;
 
-    private static final ReadCountCollection RANDOM_READ_COUNTS_WITHOUT_BAIT_BIAS;
-    private static final List<String> TRUTH_SEX_GENOTYPES_WITHOUT_BAIT_BIAS;
+    private static ReadCountCollection RANDOM_READ_COUNTS_WITHOUT_BAIT_BIAS;
+    private static List<String> TRUTH_SEX_GENOTYPES_WITHOUT_BAIT_BIAS;
 
-    static {
-        final ImmutablePair<List<String>, ReadCountCollection> randomData = generateRandomReadCountCollection(
+    @BeforeClass
+    public void before() {
+        HOMO_SAPIENS_GERMLINE_CONTIG_PLOIDY_ANNOTATIONS = ContigGermlinePloidyAnnotationTableReader
```

Comment:
This seems worse, why change these?

Reply:
I did this because the static blocks always execute (AFAICT), even if you run a single test from another package, and it calls code that emits log output:

13:15:52.512 INFO  ContigGermlinePloidyAnnotationTableReader - Ploidy tags: SEX_XX, SEX_XY

which looks misplaced depending on the context. But that is a separate problem - my main goal with this PR was to eliminate the BWA indexing, since thats super slow and it happens every time. I'm happy to revert the changes in this file for now.